app/src/main/res/values/strings.xml

```diff
@@ -95,7 +95,7 @@
 	<string name="title_transaction_preferences">Transaction Preferences</string>
 	<string name="title_account_preferences">Account Preferences</string>
 	<string name="title_default_transaction_type">Default Transaction Type</string>
-	<string name="summary_default_transaction_type">The type of transaction to use by default, CREDIT or DEBIT</string>
+	<string name="summary_default_transaction_type">CREDIT</string>
```

Comment:
This is the default summary which is shown when a default transaction type has not been set. 
The goal is to explain the setting. Once set, the currently set value will be displayed